kedro/templates/project/{{ cookiecutter.repo_name }}/README.md

```diff
@@ -11,7 +11,7 @@ Take a look at the [Kedro documentation](https://kedro.readthedocs.io) to get st
 In order to get the best out of the template:
 
 * Don't remove any lines from the `.gitignore` file we provide
-* Make sure your results can be reproduced by following a [data engineering convention](https://kedro.readthedocs.io/en/stable/11_faq/01_faq.html#what-is-data-engineering-convention)
+* Make sure your results can be reproduced by following a [data engineering convention](https://kedro.readthedocs.io/en/stable/12_faq/01_faq.html?#what-is-data-engineering-convention)
```

Comment:
I don’t think the `?` is required - it’s a anchor, not a query parameter

Reply:
Good point, removed!